src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -14,7 +14,7 @@
 public class UserPrefs implements ReadOnlyUserPrefs {
 
     private GuiSettings guiSettings = new GuiSettings();
-    private Path addressBookFilePath = Paths.get("data" , "addressbook.json");
+    private Path gradPadFilePath = Paths.get("data" , "addressbook.json");
```

Comment:
Should we update the default pathname to `gradpad.json` or something along those lines?

Reply:
Not sure if that will affect the storage part but okay I'll change it first